plugin/mouse/mouse.py

```diff
@@ -84,13 +90,14 @@ def mouse_sleep():
 
     def copy_mouse_position():
         """Copy the current mouse position coordinates"""
-        position = ctrl.mouse_pos()
-        clip.set_text(repr(position))
+        x = actions.mouse_x()
+        y = actions.mouse_y()
```

Comment:
```suggestion
        x, y = actions.mouse_x(), actions.mouse_y()
```

or
```
x, y = ctrl.mouse_pos()
```

Reply:
Could add a mouse_pos() action - I think getting the tuple is useful

Reply:
We could. I saw that we're actually calling ctrl.mouse_pos in multiple files. I would could make a separate pr changing all of those?